pkg/network/network.go

```diff
@@ -141,6 +143,13 @@ func Render(conf *cnao.NetworkAddonsConfigSpec, manifestDir string, openshiftNet
 	}
 	objs = append(objs, o...)
 
+	// render Monitoring Service
+	o, err = monitoring.RenderMonitoring(manifestDir, clusterInfo.MonitoringAvailable)
+	if err != nil {
+		return nil, err
+	}
+	objs = append(objs, o...)
```

Comment:
maybe worth to do as an else, if there is err, the `o` might be partial theoretically, 
(not good practice to use a value when there is error)

Reply:
if an error s returned from a function, the value of the artifacts is not defined ( could be partial, could be nil, depends on implementation). it is up to the function calling to it to check if `err!= nil` and only otherwise use the returned variables.

Reply:
Yes of course, this what i meant, just missed for a moment the `return`, because had in mind the monitoring is not mandatory